tests/test_responses.py

```diff
@@ -327,7 +327,7 @@ def test_head_method(test_client_factory):
 
 
 def test_empty_response(test_client_factory):
-    app = Response()
+    app = Response(status_code=200)
```

Comment:
It might be worth considering to assert either `content` or `status_code` provided, both shouldn't be empty.